server/core/src/test/java/org/infinispan/server/core/AbstractProtocolServerTest.java

```diff
@@ -43,6 +45,18 @@ public void testValidateNegativeRecvBufSize() {
       expectIllegalArgument(b, new MockProtocolServer());
    }
 
+   public void testStartingWithoutTransport() {
+      MockServerConfigurationBuilder b = new MockServerConfigurationBuilder();
+      b.startTransport(false);
+      MockProtocolServer server = new MockProtocolServer();
+      try {
+         server.start(b.build(), new DefaultCacheManager());
+         Assert.assertFalse(server.isTransportEnabled());
```

Comment:
I think it **does** make sense. Most of the changes was made in `AbstractProtocolServer` and `MockProtocolServer` derives directly from it... so why shouldn't I use it?

But fair point - I'll make it look a bit better and use `AbstractProtocolServer` as a reference.